p2p/discover/node.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/ethereum/go-ethereum/crypto/secp256k1"
 )
 
-const nodeIDBits = 512
+const NodeIDBits = 512
```

Comment:
You don't need to export this. Use `len(discover.NodeID{})` to get the length. This expression can be used as a constant.